.travis.yml

```diff
@@ -1,39 +1,22 @@
 language: java
 
 jdk:
+  - oraclejdk11
   - oraclejdk9
   - oraclejdk8
+  - openjdk11
+  - openjdk10
+  - openjdk9
   - openjdk8
-  - openjdk7
 
 matrix:
   include:
-    - env: JDK='OpenJDK 10'
-      before_install: wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
+    - jdk: openjdk7
```

Comment:
I'm not terribly familiar with TravisCI, but can you clarify why this specifies `openjdk7` but the `jdk` value above does not? 

Is this just the first value of `$TRAVIS_JDK_VERSION` used?